README.md

```diff
@@ -1 +1,65 @@
-# hello_microverse
\ No newline at end of file
+# hello_microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> One paragraph statement about the project.
+
+![screenshot](./app_screenshot.png)
+
+Additional description about the project and its features.
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
+
```

Comment:
For this built with section, you can add the languages you have used to set up this project. This will make your project even more professional. For example you can add, _HTML AND CSS_

Reply:
ok i will thank you